src/data/faq_de.json

```diff
@@ -35,6 +35,14 @@
                 "title": "Häufige Probleme",
                 "id": "most_asked",
                 "accordion": [
+                    {
+                        "title": "Firewall/Router Konfiguration: URSACHE: 4000, error during web request, HTTP status 901",
+                        "anchor": "access-list",
+                        "active": true,
+                        "textblock": [
+                             "Einige Firewalls verhindern möglicherweise den Zugriff der CWA auf das Backend. Das kann in eigenen Fällen zu einer Fehlermeldung führen, unter anderem 'URSACHE: 4000, error during web request, HTTP status 901'. Sollten sie eine Firewall benutzen (Router oder App) versichern vergewissern sie sich, dass die folgenden Domains in der Access-Liste für Web- und den DNS-Requests berücksichtigt sind:<br/>*.coronawarn.app<br/>*.t-online.de"
```

Comment:
Change eigenen to einigen

Change "benutzen (Router oder App) versichern vergewissern" to
"benutzen (Router oder App), vergewissern" (remove word versichern and add comma)
 "Einige Firewalls verhindern möglicherweise den Zugriff der CWA auf das Backend. Das kann in eigenen Fällen zu einer Fehlermeldung führen, unter anderem 'URSACHE: 4000, error during web request, HTTP status 901'. Sollten sie eine Firewall benutzen (Router oder App) versichern vergewissern sie sich, dass die folgenden Domains in der Access-Liste für Web- und den DNS-Requests berücksichtigt sind:<br/>*.coronawarn.app<br/>*.t-online.de"

Reply:
ok